.github/workflows/run-tests.yml

```
@@ -14,13 +14,15 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
         php: [8.0]
```

Let's make sure we run the checks for 3 latest php versions,

```suggestion
        php: [8.0, 8.1, 8.2]
```

---

Sorry for the delay, I have now incorporated your suggestions.